### PR TITLE
Add SPDM 1.3 new feature get_MEL for spdm_emu

### PR DESCRIPTION
**Pull request by Wenxing-hou**

### CODE REVIEW

**Comment by Wenxing-hou** [comment]
This PR will pass the CI after the lisbspdm PR(https://github.com/DMTF/libspdm/pull/2635) merge.

File: spdm_emu/spdm_emu_common/spdm_emu.c [hunk 1]
```diff
@@ -16,14 +16,16 @@ uint32_t m_exe_connection = (0 |
...
+                             EXE_CONNECTION_MEAS_MEL | 0);
 
 uint32_t m_exe_session =
     (0 | EXE_SESSION_KEY_EX | EXE_SESSION_PSK |
      /* EXE_SESSION_NO_END |*/
      EXE_SESSION_KEY_UPDATE | EXE_SESSION_HEARTBEAT | EXE_SESSION_MEAS |
      EXE_SESSION_SET_CERT | EXE_SESSION_GET_CSR |
-     EXE_SESSION_DIGEST | EXE_SESSION_CERT | EXE_SESSION_APP | 0);
+     EXE_SESSION_DIGEST | EXE_SESSION_CERT | EXE_SESSION_APP |
+     EXE_SESSION_MEAS_MEL | 0);
```
Comment (jyao1) [inline spdm_emu/spdm_emu_common/spdm_emu.c:28]: Please make it configurable by command line and update readme.

File: spdm_emu/spdm_emu_common/spdm_emu.c [hunk 2]
```diff
@@ -16,14 +16,16 @@ uint32_t m_exe_connection = (0 |
                              /* EXE_CONNECTION_VERSION_ONLY |*/
                              EXE_CONNECTION_DIGEST | EXE_CONNECTION_CERT |
                              EXE_CONNECTION_CHAL | EXE_CONNECTION_MEAS |
-                             EXE_CONNECTION_SET_CERT | EXE_CONNECTION_GET_CSR | 0);
+                             EXE_CONNECTION_SET_CERT | EXE_CONNECTION_GET_CSR |
+                             EXE_CONNECTION_MEAS_MEL | 0);
```
Comment (jyao1) [inline spdm_emu/spdm_emu_common/spdm_emu.c:20]: Please move `EXE_SESSION_MEAS_MEL` after `EXE_SESSION_MEAS`.

Comment (Wenxing-hou) [reply spdm_emu/spdm_emu_common/spdm_emu.c]: Thanks, I have changed the code.

**Pull request merged**